src/core/Akka.Remote/Transport/AkkaProtocolTransport.cs

```diff
@@ -221,17 +221,17 @@ protected override SupervisorStrategy SupervisorStrategy()
             return _supervisor;
         }
 
-#region ActorBase / ActorTransportAdapterManager overrides
+        #region ActorBase / ActorTransportAdapterManager overrides
 
         /// <summary>
         /// TBD
         /// </summary>
         /// <param name="message">TBD</param>
         protected override void Ready(object message)
         {
-            message.Match()
-                .With<InboundAssociation>(ia => //need to create an Inbound ProtocolStateActor
-                {
+            switch (message)
```

Comment:
Rewrote message handling to use `switch` rather than `PatternMatch` - went over it 2-3 times to make sure it was done correctly, guided by careful side-by-side review with JVM code and some Akka.Remote.Tests failures.